### PR TITLE
fix(ui): pod phases should be first letter capitalized

### LINKED ISSUE #2914: ui: pod phases should be first letter capitalized
## Bug Report

As the title. The problem occurs in:

https://github.com/chaos-mesh/chaos-mesh/blob/b4181f4f2ea0b8b8d6763a81d3d6f30afb369365/ui/app/src/components/NewExperimentNext/form/Scope.tsx#L44

### PR DESCRIPTION
**Pull request by g1eny0ung**
Signed-off-by: Yue Yang <g1enyy0ung@gmail.com>

### What problem does this PR solve?

Close #2914.

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [x] release-2.1
  - [x] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Review by fewdan — APPROVED** [review]
LGTM

File: ui/app/src/components/NewExperimentNext/form/Scope.tsx [hunk 1]
```diff
@@ -41,7 +42,7 @@ interface ScopeProps {
   podsPreviewDesc?: string | JSX.Element
 }
 
-const phases = ['all', 'pending', 'running', 'succeeded', 'failed', 'unknown']
+const phases = ['all', 'Pending', 'Running', 'Succeeded', 'Failed', 'Unknown']
```
Comment (cwen0) [inline ui/app/src/components/NewExperimentNext/form/Scope.tsx:45]: s/all/All/g?

Comment (g1eny0ung) [reply ui/app/src/components/NewExperimentNext/form/Scope.tsx:45]: Yes! I will fix it soon.

Comment (g1eny0ung) [reply ui/app/src/components/NewExperimentNext/form/Scope.tsx]: Already fixed.

**Review by STRRL — APPROVED** [review]

**Pull request merged**